package/requirements.txt

```
@@ -1,7 +1,8 @@
 packaging~=23.0
 kedro>=0.17.5
 ipython>=7.0.0, <9.0
-fastapi>=0.73.0, <0.96.0
+fastapi>=0.100.0,<0.104.0
```

```suggestion
fastapi>=0.73.0
```

can we just remove the upper bound